docs/index.md

```diff
@@ -1,61 +1,81 @@
 
-Tying together the tools required to release a machine learning model can be daunting. Whisk makes building and releasing ML models easy and fun. Whisk creates a logical and flexible project structure for ML with reproducible results and lets you release your model to the world without becoming a software engineer.
-
-Whisk doesn't lock you into a particular ML framework or require you to learn yet another ML packaging API. Instead, it leverages the magic of Python's ecosystem that's available to projects structured in a Pythonic-way. Whisk does the structuring while you focus on the data science.
+Whisk doesn't lock you into a particular ML framework or require you to learn yet another ML packaging API. Instead, it lets you leverage the large Python ecosystem by structuring your ML project in a [Pythonic-way](
+https://docs.python-guide.org/writing/structure/). Whisk does the structuring while you focus on the data science.
 
 Read more about our [beliefs](#beliefs).
 
-## Quickstart
+## A quick tour of whisk
+
+The best way to experience whisk is by creating your first project. The short self-guided tour below shows the power of whisk.
 
-_Replace `demo` with the name of your ML project in the examples below._
+_We use __demo__ as the project name in the examples below. If you use a different project name, be sure to replace __demo__ with the name of your project._
 
-### Create a project for the model
+### Create a project
 
-Create the project with whisk:
+Start by creating a project. Begin a terminal session and run the commands below:
 
 ```
 $ pip install whisk
-$ echo "Generate the directory structure, set up a venv, initialize a Git repo, and more."
 $ whisk create demo
 $ cd demo
 $ source venv/bin/activate
 ```
 
-Take a quick tour the project you just created:
+This installs the whisk package, creates the project, and activates the project's venv.
 
-```
-$ jupyter-notebook notebooks/getting_started.ipynb
-```
+### Try the sample model
+
+The project contains a simple model stub that returns the length of each inner list within a 2D array. The model is saved to the disk using pickle. We can use the model stub to demonstrate the full capabilities of a whisk project.
+
+Since a whisk project uses a standard Python project code structure, there are multiple ways to invoke the model.
 
-The notebook shows how to save your trained model to disk, use the saved model to generate predictions, how to load Python functions and classes from the project's `src` directory for a cleaner notebook, and more. It's the guide rails for your own ML project.
+#### From Python code
+
+The model code resides in the `src/demo` directory of the project. The source code is installed as an editable Python package, so you can run the model from the Python console or from within a notebook cell:
```

Comment:
Should `src/demo` be `src/demo/models`?

Reply:
I realize that other "model code" is sitting in `src` but in this example I want to make sure someone can quickly find the code we are referencing below

Reply:
> Should src/demo be src/demo/models

Yes we can update that. See my note below about also putting the demo project on GitHub.